{
    "config": {
        "error": {
            "cannot_connect": "Det gick inte att ansluta.",
            "invalid_auth": "Ogiltig autentisering",
            "not_supported": "Endast PrusaLink API v2 st\u00f6ds",
            "unknown": "Ov\u00e4ntat fel"
        },
        "step": {
            "user": {
                "data": {
                    "api_key": "API-nyckel",
                    "host": "V\u00e4rd"
                }
            }
        }
    }
}